lib/xcpretty/parser.rb

```diff
@@ -77,6 +77,10 @@ module Matchers
     # $2 file_name (e.g. Main.storyboard)
     COMPILE_STORYBOARD_MATCHER = /^CompileStoryboard\s(.*\/([^\/].*\.storyboard))/
 
+    # @regex Captured groups
+    # $1 file_path (e.g. Assets.xcassets)
+    COMPILE_ASSET_CATALOG_MATCHER = /^CompileAssetCatalog\s.*\s(.*\.xcassets)/
```

Comment:
If there is space in the assets catalog path the matcher will find partial path. For example 
`Test\ Host/Assets.xcassets` will find `Host/Assets.xcassets`. See how I deal with it in `COMPILE_METAL_MATCHER`.